dist/ge-1-1-6/samples/entando-solrCloud.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: solr.apache.org/v1beta1
+kind: SolrCloud
+metadata:
+  name: solr
+spec:
+  solrAddressability:
```

Comment:
@avdev4j I'm not convinced on this section. I thought we did not want any external ingress, both because it's not needed and we don't want to expose it anyway. If we drop it, then there are no placeholders in this file to be manually edited. CC: @jyunmitch

Reply:
The related documentation is not totally clear about that but I'm pretty it can't be removed to not exposing outside.
https://apache.github.io/solr-operator/docs/solr-cloud/solr-cloud-crd.html#addressability

That would mean remove the entire `solrAddressability:` section

Reply:
@avdev4j You think it "can" be removed?

Reply:
oups, edited. Yes it can be removed ;)